vnext/Desktop.IntegrationTests/HttpResourceIntegrationTests.cpp

```diff
@@ -11,34 +11,35 @@ using namespace Microsoft::VisualStudio::CppUnitTestFramework;
 using std::string;
 using std::vector;
 
-TEST_CLASS(HttpResourceIntegrationTest){TEST_METHOD(MakeIsNotNull){auto rc = IHttpResource::Make();
-Assert::IsFalse(nullptr == rc);
-}
-
-TEST_METHOD(RequestGetSucceeds) {
-  auto rc = IHttpResource::Make();
-  bool sent = false;
-  bool received = false;
-  string error;
-  rc->SetOnRequest([&sent]() { sent = true; });
-  rc->SetOnResponse([&received](const string &message) { received = true; });
-  rc->SetOnError([&error](const string &message) { error = message; });
-
-  rc->SendRequest("GET", "http://localhost:8081/debugger-ui", {}, dynamic(), "text", false, 1000, [](int64_t) {});
-
-  Assert::IsTrue(sent);
-  Assert::IsTrue(received);
-  Assert::AreEqual(string(), error);
-}
-
-TEST_METHOD(RequestGetFails) {
-  auto rc = IHttpResource::Make();
-  string error;
-  rc->SetOnError([&error](const string &message) { error = message; });
-
-  rc->SendRequest("GET", "http://nonexistinghost", {}, dynamic(), "text", false, 1000, [](int64_t) {});
-
-  Assert::AreEqual(string("No such host is known"), error);
-}
-}
-;
+TEST_CLASS (HttpResourceIntegrationTest) {
```

Comment:
Are we still fighting against clang-format in these files? You can add a local .clang-format file to disable it for the directory if we're still having issues.

Reply:
I have changed the .clang-format in tis change to use TEST_CLASS macro as a 'FOR_EACH'-like macro. It effectively fixed a lot of previous formatting issues in our unit tests.

---
In reply to: [374993626](https://github.com/microsoft/react-native-windows/pull/4030#discussion_r374993626) [](ancestors = 374993626)